tests/integration/run_tests.rs

```diff
@@ -860,7 +860,7 @@ fn lock_no_declaration_files() {
     .build();
   let output = context
     .new_command()
-    .args("cache --lock --lock-write $TESTDATA/lockfile/no_dts/main.ts")
+    .args("cache --lock deno.lock $TESTDATA/lockfile/no_dts/main.ts")
```

Comment:
Not sure about this one. It seems strange that `--lock` without an argument doesn't work.